### ISSUE TITLE
Fatal error when adding lifecycle widget to dashboard

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#8821]
## Bug Description
When testing adding widgets to the dashboard, as soon as I select the contacts lifecycle I see an error on the screen behind the modal pop-up, then when I save my changes I see an error in the modal itself.

```
500 Internal Server Error - Type error: Argument 2 passed to Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType::__construct() must be an instance of Recurr\Transformer\TranslatorInterface, instance of Symfony\Component\Translation\DataCollectorTranslator given, called in /var/app/current/code/pulls/3.x/var/cache/dev/Container4xcpg7u/getMautic_Form_Type_LeadDashboardLeadsLifetimeWidgetService.php on line 8

Symfony\Component\Debug\Exception\FatalThrowableError
/app/bundles/LeadBundle/Form/Type/DashboardLeadsLifetimeWidgetType.php:32 at
/var/cache/dev/Container4xcpg7u/getMautic_Form_Type_LeadDashboardLeadsLifetimeWidgetService.php:8 at Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType -> __construct ( object(ListModel), object(DataCollectorTranslator) )
/var/cache/dev/Container4xcpg7u/appDevDebugProjectContainer.php:5831 at require ( '/var/app/current/code/pulls/3.x/var/cache/dev/Container4xcpg7u/getMautic_Form_Type_LeadDashboardLeadsLifetimeWidgetService.php' )
/var/cache/dev/Container4xcpg7u/appDevDebugProjectContainer.php:8570 at Container4xcpg7u\appDevDebugProjectContainer -> load ( 'getMautic_Form_Type_LeadDashboardLeadsLifetimeWidgetService.php' )
/vendor/symfony/dependency-injection/ServiceLocator.php:64 at Container4xcpg7u\appDevDebugProjectContainer -> Container4xcpg7u\{closure} ( )
/vendor/symfony/form/Extension/DependencyInjection/DependencyInjectionExtension.php:64 at Symfony\Component\DependencyInjection\ServiceLocator -> get ( 'Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType' )
/vendor/symfony/form/FormRegistry.php:78 at Symfony\Component\Form\Extension\DependencyInjection\DependencyInjectionExtension -> getType ( 'Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType' )
/vendor/symfony/form/FormFactory.php:74 at Symfony\Component\Form\FormRegistry -> getType ( 'Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType' )
/vendor/symfony/form/FormFactory.php:38 at Symfony\Component\Form\FormFactory -> createNamedBuilder ( 'params', 'Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType', null, array('label' => false, 'auto_initialize' => false) )
/vendor/symfony/form/Form.php:839 at Symfony\Component\Form\FormFactory -> createNamed ( 'params', 'Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType', null, array('label' => false, 'auto_initialize' => false) )
/app/bundles/DashboardBundle/Form/Type/WidgetType.php:151 at Symfony\Component\Form\Form -> add ( 'params', 'Mautic\LeadBundle\Form\Type\DashboardLeadsLifetimeWidgetType', array('label' => false, 'auto_initialize' => false) )
/vendor/symfony/event-dispatcher/EventDispatcher.php:214 at Mautic\DashboardBundle\Form\Type\WidgetType -> Mautic\DashboardBundle\Form\Type\{closure} ( object(FormEvent), 'form.pre_bind', object(EventDispatcher) )
/vendor/symfony/event-dispatcher/EventDispatcher.php:44 at Symfony\Component\EventDispatcher\EventDispatcher -> doDispatch ( array(array(object(TrimListener), 'preSubmit'), array(object(CsrfValidationListener), 'preSubmit'), object(Closure)), 'form.pre_bind', object(FormEvent) )
/vendor/symfony/event-dispatcher/ImmutableEventDispatcher.php:33 at Symfony\Component\EventDispatcher\EventDispatcher -> dispatch ( 'form.pre_bind', object(FormEvent) )
/vendor/symfony/form/Form.php:552 at Symfony\Component\EventDispatcher\ImmutableEventDispatcher -> dispatch ( 'form.pre_bind', object(FormEvent) )
/vendor/symfony/form/Extension/HttpFoundation/HttpFoundationRequestHandler.php:109 at Symfony\Component\Form\Form -> submit ( array('name' => 'Lifecycle', 'type' => 'lead.lifetime', 'width' => '25', 'height' => '215', 'id' => '', '_token' => 'X7sIODusP62rBmDF72KUFmBpyC-am0n2I9HUlLficRc', 'buttons' => array('save' => '')), true )
/vendor/symfony/form/Form.php:487 at Symfony\Component\Form\Extension\HttpFoundation\HttpFoundationRequestHandler -> handleRequest ( object(Form), object(Request) )
/app/bundles/CoreBundle/Controller/AbstractFormController.php:167 at Symfony\Component\Form\Form -> handleRequest ( object(Request) )
/app/bundles/DashboardBundle/Controller/DashboardController.php:111 at Mautic\CoreBundle\Controller\AbstractFormController -> isFormValid ( object(Form) )
/app/bundles/CoreBundle/Controller/CommonController.php:449 at Mautic\DashboardBundle\Controller\DashboardController -> newAction ( '0', '' )
/vendor/symfony/http-kernel/HttpKernel.php:151 at Mautic\CoreBundle\Controller\CommonController -> executeAction ( 'new', '0', '0', '' )
/vendor/symfony/http-kernel/HttpKernel.php:68 at Symfony\Component\HttpKernel\HttpKernel -> handleRaw ( object(Request), '1' )
/vendor/symfony/http-kernel/Kernel.php:200 at Symfony\Component\HttpKernel\HttpKernel -> handle ( object(Request), '1', true )
/app/AppKernel.php:104 at Symfony\Component\HttpKernel\Kernel -> handle ( object(Request), '1', true )
/app/middlewares/CORSMiddleware.php:91 at AppKernel -> handle ( object(Request), '1', true )
/app/middlewares/CatchExceptionMiddleware.php:43 at Mautic\Middleware\CORSMiddleware -> handle ( object(Request), '1', true )
/app/middlewares/Dev/IpRestrictMiddleware.php:64 at Mautic\Middleware\CatchExceptionMiddleware -> handle ( object(Request), '1', true )
/app/middlewares/VersionCheckMiddleware.php:67 at Mautic\Middleware\Dev\IpRestrictMiddleware -> handle ( object(Request), '1', true )
/app/middlewares/TrustMiddleware.php:51 at Mautic\Middleware\VersionCheckMiddleware -> handle ( object(Request), '1', true )
/vendor/stack/builder/src/Stack/StackedHttpKernel.php:23 at Mautic\Middleware\TrustMiddleware -> handle ( object(Request), '1', true )
/vendor/stack/run/src/Stack/run.php:13 at Stack\StackedHttpKernel -> handle ( object(Request) )
/index_dev.php:25 at Stack\run ( object(StackedHttpKernel) )
/index.php:6 at require_once ( '/var/app/current/code/pulls/3.x/index_dev.php' )
```

| Q   | A
| --- | ---
| Mautic version | 3.0.0-beta-2
| PHP version | 7.2.24 (Mautibox)
| Browser | Chrome

### Steps to reproduce
1. Add Lifecycle widget to dashboard

**Comment by RCheesley** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Dashboard > Add contact widgets to dashboard](https://trello.com/c/nQHtDkbS/14-dashboard-add-contact-widgets-to-dashboard)

**Comment by kuzmany** [comment]
I'll take it.

**Comment by kuzmany** [comment]
Fix https://github.com/mautic/mautic/pull/8846

**Comment by dennisameling** [comment]
Fixed by #8825
